src/java.base/share/classes/java/security/PrivilegedActionException.java

```diff
@@ -98,21 +98,25 @@ public String toString() {
     /**
      * Serializable fields for UndeclaredThrowableException.
      *
-     * @serialField undeclaredThrowable Throwable
+     * @serialField exception Exception the undeclaredThrowable
```

Comment:
The "undeclaredThrowable" is confusing and not defined anywhere in this class specification. Suggest changing this to: 

The exception thrown by the privileged computation that resulted in this {@code PrivilegedActionException}.

Reply:
Sure, good idea.  Thanks.